shell/platform/fuchsia/flutter/flatland_connection.cc

```diff
@@ -81,6 +126,44 @@ void FlatlandConnection::DoPresent() {
   // the correct ones for VulkanSurface's interpretation.
   previous_present_release_fences_.clear();
   previous_present_release_fences_.swap(current_present_release_fences_);
+  previous_release_overflow_ = current_release_overflow_;
+  current_release_overflow_ = nullptr;
+
+  // Similar to the treatment of acquire_fences_overflow_ above. Except in
+  // the other direction.
+  if (previous_release_overflow_ != nullptr) {
+    FML_CHECK(previous_release_overflow_->event_.is_valid());
+
+    std::shared_ptr<Overflow> fences = previous_release_overflow_;
+
+    async::PostTask(dispatcher_, [dispatcher = dispatcher_,
+                                  fences = previous_release_overflow_]() {
+      FML_CHECK(fences != nullptr);
+      FML_CHECK(fences->event_.is_valid());
+
+      auto wait = std::make_unique<async::WaitOnce>(fences->event_.release(),
```

Comment:
Same as above. I think we should use get() instead.

Reply:
Done.